src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -35,9 +39,14 @@ public override IDisplayResult Display(ListPart listPart, BuildPartDisplayContex
                     {
                         var pager = await GetPagerSlimAsync(context);
                         var settings = context.TypePartDefinition.GetSettings<ListPartSettings>();
+                        var containedItemOptions = new ContainedItemOptions();
+                        containedItemOptions.Status = ContentsStatus.Published;
```

Comment:
It might be more convenient to make `Published` the default, and drop the `None`

It's what you want to default for the front end when you need to reuse the service.

I will let you decide

Reply:
On the admin you need all the latest items though. Maybe `None` could be explicitly `Latest`?

Reply:
My concern is this service is used for both front end (and decoupled) and the admin. So I think Published is safest as the default. You can’t go wrong then without intentionally going wrong

Reply:
Made `Published` the default, `None` was removed earlier.